packages/store/src/decorator-factory.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import { Store } from './store';
+
+/**
+ * Allows decorator to get access to the DI store.
+ * @ignore
+ */
+@Injectable()
+export class DecoratorFactory {
```

Comment:
We already have a select factory we can just rename to be more generic use that.

Reply:
Okay, now I should to explain what is decorator factory. 

In short, it's just refactoring.

Decorator factory is refactored selector factory which was moved to own file service file. Why? Because right now, decorator factory provides service for selector and dispatch decorators.